### ISSUE TITLE
Support for openff-bespokefit

### ISSUE THREAD

**Issue by jmichel80** [OpenBioSim/biosimspace#262]
It would be desirable to make it easier for users to combine molecules parameterised with openff-bespokefit with other system components.  This would lower barriers for benchmarking studies with forcefield parameters optimised for each ligand which should be considered the current SOTA for classical force field FEP studies. 

This could be done by wrapping around openff code described [here](https://github.com/openforcefield/2023-workshop-vignettes/blob/main/G-PDB-bespokefit-to-simulation.ipynb) and extending the[ BioSimSpace.Parameters API](https://biosimspace.openbiosim.org/api/index_Parameters.html) 

At present interchange may be used to export a parameterised molecule into a file format that is compatible with a Sire parser. Longer term a direct openmm->sire conversion may be more robust.

**Comment by bcossins** [comment]
+1

**Comment by lohedges** [comment]
I looked at this but there appears to be a mind boggling amount of options etc. for bespoke fit. I imagine we wouldn't need all of these, but it might limit power users. 

What's the issue with a user writing their OpenFF object to AMBER format and loading? Is this lossy, in which case direct conversion would be the only route.

**Comment by jmichel80** [comment]
Maybe that works. Users seem unsure how to implement this feature themselves. Maybe all that is needed is to document a pipeline showing how to run BFE calculations with bespoke ffs.

**Comment by lohedges** [comment]
Yes, I think some examples would help. I guess people might be unaware that you can mix things in BioSimSpace, i.e. you don't need everything to be in one topology file: you can create multiple molecules in different ways, then add them together.

**Comment by bcossins** [comment]
That might be one of the reasons to wrap it. Cos it is so complex to work
it out. You could parameterise any of the bits of your system outside of
BSS but it is convenient to wrap simplified versions.

**Comment by lohedges** [comment]
I agree with that, my issue was more that there didn't (yet) appear to be a "standard" protocol to follow, so it wasn't clear what options to expose and what generally appropriate settings would be.

**Comment by bcossins** [comment]
Might be able to come up with a default one soon. Will come back on that.

**Comment by lohedges** [comment]
Great. I haven't looked for quite a while so it might be that things are more standardised now.

**Comment by bcossins** [comment]
Not sure about that but there is a fast charge aware NNP which makes it
easier as things run faster
